app/controllers/alchemy/messages_controller.rb

```diff
@@ -124,7 +124,7 @@ def get_page
     end
 
     def message_params
-      params.require(:message).permit(*mailer_config["fields"])
+      params.require(:message).permit(*mailer_config["fields"], :contact_form_id)
```

Comment:
Would be nice to have a spec for this, so we are sure the bug is truly fixed.

Reply:
Added 2 specs